### PR TITLE
License cleanup help

### PR DESCRIPTION
**Pull request by siwinski**
I am not a lawyer or a member of Red Hat's legal team, but I noticed that the license information in this package is off.  This pull request is an attempt to help clean that up based on my knowledge of other open source packages.

Changes:
- Change license file name from `OFL.md` to `LICENSE.md` because this package is dual-licensed (not just OFL) and this is the standard license file name to use
- Update README to link to the license file instead of specifying 1 of 2 licenses (and no need to list the text twice)
- Fix `package.json` license value
    - https://docs.npmjs.com/files/package.json#license
    - https://spdx.org/licenses/

Questions I still have about this package:
- Is the LGPL licensing really just 2.1 or is it 2.1+?
- Did Red Hat legal review and approve both the licensing of this package and the format of the license file?

@starryeyez024 @rethomps @castastrophe

### CODE REVIEW

**Comment by siwinski** [comment]
The README file also references the "SIL license" twice in its' description.  Is this repo licensed under a third license as well?

**Comment by starryeyez024** [comment]
per this question:
Did Red Hat legal review and approve both the licensing of this package and the format of the license file?

Yes. I did send them this PR though.

File: package.json [hunk 1]
```diff
@@ -12,5 +12,5 @@
     "Hat"
   ],
   "author": "",
-  "license": "SEE LICENSE IN OFL.md"
+  "license": "(OFL-1.1 OR LGPL-2.1)"
```
Comment (davelab6) [inline package.json]: OFL-1.1 only please :)

**Comment by richardfontana** [comment]
@starryeyez024 @siwinski This should be closed in favor of the in-progress #7 - the main difference is that this keeps the OFL|LGPL dual license (even though @siwinski rightly found that peculiar) and @davelab6 is right that, at least since we're trying to fix this up anyway, we shouldn't have an LGPL license option because it's just confusing at best and no real-world user will benefit from such an option.

**Comment by siwinski** [comment]
Closing per last comment.  Thanks @richardfontana !

**Comment by davelab6** [comment]
Thanks!

**Pull request closed without merging**